pytorch_lightning/accelerators/cpu.py

```diff
@@ -62,3 +62,8 @@ def auto_device_count() -> int:
     def is_available() -> bool:
         """CPU is always available for execution."""
         return True
+
+    @staticmethod
+    def hardware_name() -> str:
```

Comment:
I would prefer the name `accelerator_type`

Reply:
> if we want this to work with auto, then it should be a abstractmethod staticmethod on the base interface.

Absolutely! It should be part of base class as abstractmethod as well.

> i prefer name here to reduce verbosity

We have `strategy_name` attribute as part of Strategy. I think we could do `accelerator_name` as well.

Reply:
can we rename `strategy_name` to be simply `name` ? `strategy_name` was only recently introduced in the accelerator connector rewrite, so it's safe to change immediately w/o concerns of breaking backward compatibility. @four4fish